pkgs/by-name/et/etterna/package.nix

```
@@ -0,0 +1,119 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  fetchgit,
+  writeShellScript,
+  makeDesktopItem,
+  copyDesktopItems,
+  # deps
+  clang,
+  cmake,
+  python2,
+  pkg-config,
+  openssl,
+  libGLU,
+  xorg,
+  alsa-lib,
+  libjack2,
+  libpulseaudio,
+  libogg,
+
+  # cfgs
+  enableCrashpad ? false,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "etterna";
+  version = "0.74.3";
+
+  src = fetchFromGitHub {
+    owner = "etternagame";
+    repo = "etterna";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-zzCk6axISswfTAk7rRha5HFzIHQ0AjpAZyAWzH+Cn1s=";
+  };
+
+  patches = [ ./fix-download-manager.patch ];
+
+  nativeBuildInputs = [
+    clang
+    cmake
+    pkg-config
+
+    openssl
+
+    alsa-lib
+    libjack2
+    libpulseaudio
+
+    libGLU
+    libogg
+
+    xorg.libXinerama
+    xorg.libXrandr
+    xorg.libX11
+    xorg.libXext # Needed for DPMS
+    xorg.libXvMC
+
+    copyDesktopItems
+  ] ++ lib.optionals enableCrashpad [ python2 ];
+
+  desktopItems = [
+    (makeDesktopItem {
+      name = "etterna";
+      desktopName = "Etterna";
+      genericName = "Rhythm and dance game";
+      icon = "etterna";
+      tryExec = "etterna";
+      exec = "etterna";
+      categories = [
+        "Application"
+        "Game"
+        "ArcadeGame"
+      ];
+      comment = "A cross-platform rhythm video game.";
+      terminal = false;
+    })
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/{bin,share/etterna}
+    mkdir -p $out/share/applications
+    # copy select necessary game files into virtual fs
+    for dir in \
+      Announcers Assets BGAnimations \
+      BackgroundEffects BackgroundTransitions \
+      Data GameTools NoteSkins Scripts \
+      Themes
+    do
+      cp -r "/build/source/$dir" "$out/share/etterna/$dir"
+    done
```

```suggestion
    cp -r /build/source/{Announcers,Assets,BGAnimations,BackgroundEffects,BackgroundTransitions,Data,GameTools,NoteSkins,Scripts,Themes} $out/share/etterna
```
Could check the contents of `/build/source/` to see if this can be simplified